requirements.txt

```diff
@@ -16,4 +16,5 @@
 six>=1.8.0
 appdirs
 enum-compat
+SpiNNUtilities >= 1!5.1.1, < 1!6.0.0
```

Comment:
While I don't think this is an issue better check with @lplana that he does not mind a new dependency for spalloc!

Reply:
> While I don't think this is an issue better check with @lplana that he does not mind a new dependency for spalloc!

No objection from me.

Reply:
This isn't for the server, this is for the client.